test/extensions/filters/http/ext_authz/ext_authz_test.cc

```diff
@@ -203,7 +204,7 @@ TEST_F(HttpFilterTest, ErrorCustomStatusCode) {
     envoy_grpc:
       cluster_name: "ext_authz_server"
   failure_mode_allow: false
-  status_on_error: 
```

Comment:
do we need to have spaces in end of lines?

Reply:
No, I think superfluous space should be cleaned up. Thanks!